common/culture/cultures/ogre.txt

```diff
@@ -2,15 +2,12 @@
 	graphical_cultures = {
```

Comment:
Do you plan to add Mok'nathal here or on another branch?

Reply:
> Do you plan to add Mok'nathal here or on another branch?

Haven't thought about it yet. I suppose I could just add them to this one.